README.md

```diff
@@ -26,16 +26,17 @@ libp2p-switch is used by [libp2p](https://github.com/libp2p/js-libp2p) but it ca
 - [Usage](#usage)
   - [Create a libp2p switch](#create-a-libp2p-switch)
 - [API](#api)
-  - [`switch.dial(peer, protocol, callback)`](#swarmdialpi-protocol-callback)
-  - [`switch.hangUp(peer, callback)`](#swarmhanguppi-callback)
-  - [`switch.handle(protocol, handler)`](#swarmhandleprotocol-handler)
-  - [`switch.unhandle(protocol)`](#swarmunhandleprotocol)
-  - [`switch.start(callback)`](#swarmlistencallback)
-  - [`switch.stop(callback)`](#swarmclosecallback)
-  - [`switch.connection`](#connection)
+  - [`switch.dial(peer, protocol, callback)`](#switchdialpeer-protocol-callback)
+  - [`switch.dialFSM(peer, protocol, callback)`](#switchdialfsmpeer-protocol-callback)
```

Comment:
better just mimic js-libp2p and do dial and dialProtocol

Reply:
We don't need to block this PR for this. What I mean is that given that libp2p (the module) is just an extension to the libp2p-switch (the dialing machine) that adds it other features, it would be cool if they both expose the same dialing/hangup APIs.

Reply:
That makes sense. Ideally I'd like to deprecate `dialProtocol` in libp2p and just use `dial`. Having both feel unnecessary. I'll propose the change in an issue there and if `dialProtocol` ends up staying we can add that in here.